CHANGELOG.md

```diff
@@ -40,771 +39,784 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
-* (rpc) [#1720](https://github.com/evmos/ethermint/pull/1720) Fix next block fee for historical block and calculate base fee by params.
```

Comment:
https://www.diffchecker.com/eVItH9j1

Reply:
sync with what?

Reply:
to compare with [main](https://github.com/evmos/ethermint/blob/main/CHANGELOG.md?plain=1) branch, since changelog broke due to some backport changes

Reply:
our changelogs is maintained with the backports, why compare with ethermint main?

Reply:
oh, I see